tests/complex/fake_rhevm4.py

```diff
@@ -31,10 +51,19 @@ def check_version_header(self):
 
 
 class FakeRhevm4(FakeVirt):
+
+    virt_type = "rhevm"
+
     def __init__(self, port=None):
         super(FakeRhevm4, self).__init__(Rhevm4Handler, port=port)
         self.server._data_version = self._data_version
 
+
 if __name__ == '__main__':
-    rhevm = FakeRhevm4(port=8443)
+    if len(sys.argv) >= 2:
+        port = int(sys.argv[1])
+    else:
+        port = None
+    rhevm = FakeRhevm4(port=port)
+    print(rhevm.virt_who_config())
```

Comment:
Why print this? For debug reasons?

Reply:
It is part of smoke test. When you want to test virt-who against this fake server, then it is good to see the content of configuration file that could be used for testing.